var/spack/repos/builtin/packages/itk/package.py

```diff
@@ -58,13 +58,18 @@ class Itk(CMakePackage):
     depends_on("expat")
     depends_on("fftw-api")
     depends_on("googletest")
-    depends_on("hdf5+cxx")
+    depends_on("hdf5+cxx+hl~mpi")
     depends_on("jpeg")
     depends_on("libpng")
     depends_on("libtiff")
-    depends_on("mpi")
     depends_on("zlib-api")
 
+    patch(
+        "https://patch-diff.githubusercontent.com/raw/InsightSoftwareConsortium/ITK/pull/4041.diff?full_index=1",
```

Comment:
Would it be possible to use the commit on the master branch instead of the diff in the PR? Super small nitpick but it makes me nervous to point at an external PR instead of the mainline tree.

Reply:
@alecbcs fixed